modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -81,20 +92,22 @@ Scan for additional .ini files in: /usr/local/etc/php/7.4/conf.d
 Additional .ini files parsed:      (none)
 ----
 +
-2. Insert a line in the php.ini file specifying the extension to be loaded; this should be in the `[PHP]` section.
+2. Insert a line in the `php.ini` file specifying the extension to be loaded; this should be in the `[PHP]` section.
 If you don't know where that is, simply search for existing commented or uncommented `extension=` entries in the file.
 Note that the line differs for Linux/Mac OS from Microsoft Windows.
 
+> Note that, on PHP version 7.2.0 and up, simply using the extension name is preferred.
+
 .Mac & Linux
-[source,toml]
+[source,ini]
 ----
-extension=couchbase.so
+extension=couchbase
```

Comment:
This is related the the comment on 7.2.0 above?

It does mean that we don't need 2 separate sections for Mac/Linux + Windows.
(As 7.2 is EOL, that incantation should work for everyone now?)

Reply:
Yes. The only reservation I have is that I discovered this first in the latest `php.ini` and then later verified it in the Windows section of the PHP documentation. However, even there, it states it's "OS-independent" so I think we're safe with this suggestion across the board, but I only tested like this on Windows.

> Note that, on PHP version 7.2.0 and up, the extension name may be used instead of the extension's file name. As this is OS-independent and easier, especially for newcomers, it becomes the recommended way of specifying extensions to load. File names remain supported for compatibility with prior versions.

https://www.php.net/manual/en/install.pecl.windows.php#install.pecl.windows.loading